wsi/overlaybuffer.h

```diff
@@ -50,7 +50,8 @@ class OverlayBuffer {
 
   virtual uint32_t GetFb() const = 0;
 
-  virtual GpuImage ImportImage(GpuDisplay egl_display) = 0;
```

Comment:
That would require each of these Gpu* types to be a union of all possible types and we would have to add a flag somewhere to declare what type is being used. If we want runtime selection to work, then ImportImage() needs to act differently depending on what backend is calling it. I chose to use function overrides because it seemed a little cleaner.